.travis.yml

```diff
@@ -19,14 +19,32 @@ env:
         - COASTLINEDIR="$HOME/gmt-install-dir/coast"
         - PATH="$INSTALLDIR/bin:$PATH"
         - LD_LIBRARY_PATH="$INSTALLDIR/lib:$LD_LIBRARY_PATH"
+        - TRAVIS_EVENT_TYPE=cron
+        - if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then
+            export PYTHON=3.6
+            export CONDA_REQUIREMENTS="ci/requirements.txt"
+            export BUILD_DOCS=true
+          fi
 
 before_install:
     # Install GMT dependencies
     - bash ci/travis-setup.sh
+    - if [ "$BUILD_DOCS" == "true" ]; then
+        # Install GMT documentation dependencies
+        # Get the Fatiando CI scripts
+        git clone https://github.com/fatiando/continuous-integration.git
+        # Download and install miniconda and setup dependencies
+        # Need to source the script to set the PATH variable globaly
+        source continuous-integration/travis/setup-miniconda.sh
+      fi
 
 install:
     # Build and install GMT
     - bash ci/travis-build.sh;
+    - if [ "$BUILD_DOCS" == "true" ]; then
+        # Build GMT documentations
+        bash ci/travis-build-docs.sh;
```

Comment:
Do we need a script for this?

Reply:
The build should probably be part of the `script` block as well since this is pretty much part of the tests.